python/paddle/fluid/executor.py

```diff
@@ -432,12 +432,12 @@ def handler(self, res_dict):
 class Executor(object):
     """
     An Executor in Python, supports single/multiple-GPU running,
-    and single/multiple-CPU running. When construction the Executor,
-    the device is required.
+    and single/multiple-CPU running.
 
     Args:
-        place(fluid.CPUPlace()|fluid.CUDAPlace(n)): This parameter represents
-            the executor run on which device.
+        place(fluid.CPUPlace()|fluid.CUDAPlace(n)|None): This parameter represents
+            the executor run on which device. When this parameter is None, Paddle
+	    would choose the device independtantly. Default is None.
```

Comment:
"choose the device independtantly" doesn't talk explicitly the behavior. More details is needed here.

Reply:
Done.